cuda_core/cuda/core/experimental/_device.pyx

```diff
@@ -1080,7 +1080,7 @@ class Device:
         if self._uuid is None:
             dev = self._id
             with nogil:
-                IF CUDA_CORE_BUILD_MAJOR == "12":
+                IF CUDA_CORE_BUILD_MAJOR == 12:
```

Comment:
This seems like a meaningful change. Was it just silently failing or testing false before?